### PR TITLE
#137 DrawableMatcher update

### PR DESCRIPTION
**Pull request by ahampe**
Adjusted `DrawableMatcher` to use drawable argument as expected drawable instead of actual. Added tests to sample to confirm that behavior is as expected.

### CODE REVIEW

File: sample/src/androidTest/kotlin/com/agoda/sample/DrawableListTest.kt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                                R.drawable.ic_sentiment_very_satisfied_black_24dp,
+                                appContext.theme
+                        )
+                        hasDrawable(drawable!!)
+                    }
+                }
+            }
+        }
+    }
+}
```
Comment (Unlimity) [inline sample/src/androidTest/kotlin/com/agoda/sample/DrawableListTest.kt]: Please don't forget the new line at the end of the file

File: sample/src/androidTest/kotlin/com/agoda/sample/screen/DrawableListScreen.kt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+class DrawableListScreen : Screen<DrawableListScreen>() {
+    val list = KAbsListView(
+            builder = { withId(R.id.drawableList) },
+            itemTypeBuilder = { itemType(::Item) })
+
+    class Item(i: DataInteraction) : KAdapterItem<Item>(i) {
+        val imageView = KImageView(i) { withId(R.id.imgView) }
+    }
+}
```
Comment (Unlimity) [inline sample/src/androidTest/kotlin/com/agoda/sample/screen/DrawableListScreen.kt]: Please don't forget the new line at the end of the file

File: sample/src/main/res/layout/activity_drawable_list.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ListView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/drawableList"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" />
```
Comment (Unlimity) [inline sample/src/main/res/layout/activity_drawable_list.xml]: Please don't forget the new line at the end of the file

**Comment by ahampe** [comment]
Apologies for the delay; hectic life stuff and whatnot. The newlines are now in place.

**Comment by Unlimity** [comment]
Merged. Thank you for your contribution!

**Pull request merged**